lib/network_manager.py

```diff
@@ -0,0 +1,26 @@
+import json
+import logging
+
+logging.basicConfig(format='%(asctime)s | %(name)s | %(message)s',
+                    level=logging.DEBUG)
+
+
+data_file = '../networks_info/nue_ses_network.json'
```

Comment:
What is this related to? Current file or directory?

Reply:
predefined path removed